docs/reference/driver-compatibility.txt

```diff
@@ -243,7 +243,7 @@ for that Ruby version is deprecated.
      - |checkmark|
      - |checkmark|
      - D
-     - D
+     -
```

Comment:
Can this PR go into 2.16 rather than 2.15?  Did we specifically intend to remove this support in 2.15?

Reply:
I think the idea is to remove 2.3 support in the next release. I assumed it would be 2.15.

Reply:
We've already had 2.15 alpha and 2.15 has many other changes (i.e. all of the 5.0). Plus it's late in the dev cycle. I would suggest leaving Ruby version bump to 2.16, either by holding off on merging this PR or by branching 2.15-stable prior to the merge (holding off on merging this PR should be less work overall).